Bump Katello to 4.13.0 and related packages to newest versions

qcjames53 opened the pull request:
This work was completed as a part of Katello 4.12 branching. Other than 'katello-host-tools' (which needed to be updated to version 4.3.0), all packages were already on the latest versions.

chris1984 reviewed (approved):
Looks good to me, @evgeni can this get a review?

qcjames53 commented:
Any ideas what is going on in the build playbook for katello-host-tools? It looks like it's trying to build 4.2.3 even though I updated the version number to 4.3.0. I double checked and the gh links are all active and ok.

evgeni commented:
> Any ideas what is going on in the build playbook for katello-host-tools? It looks like it's trying to build 4.2.3 even though I updated the version number to 4.3.0. I double checked and the gh links are all active and ok.

You bumped the version in the spec, but didn't remove the 4.2.3 version from git-annex, nor added 4.3.0 to it.

But even if you do, it won't just build as seen here: https://github.com/theforeman/foreman-packaging/pull/9435

I'd suggest dropping k-h-t from this PR and follow up on the other one.

Also, someone needs to bump hammer_cli_katello, no preference if here or in an own PR.

evgeni commented on packages/katello/rubygem-katello/rubygem-katello.spec:
```diff
@@ -5,8 +5,8 @@
 %global foreman_max_version 3.11
```
you need to bump this (and the line above)

qcjames53 replied:
Will do, thanks. That's interesting we're bumping the minimum to 3.11.

qcjames53 commented:
> > Any ideas what is going on in the build playbook for katello-host-tools? It looks like it's trying to build 4.2.3 even though I updated the version number to 4.3.0. I double checked and the gh links are all active and ok.
> 
> You bumped the version in the spec, but didn't remove the 4.2.3 version from git-annex, nor added 4.3.0 to it.
> 
> But even if you do, it won't just build as seen here: #9435
> 
> I'd suggest dropping k-h-t from this PR and follow up on the other one.
> 
> Also, someone needs to bump hammer_cli_katello, no preference if here or in an own PR.

Thanks for the help with this; I didn't know about the build failures with 4.3 from the other PR. I dropped katello-host-tools from this PR. We can add hammer katello as a PR for release.

ianballou commented:
This PR mirrors what I did during 4.11 branching yet the build is failing -- did something change?

Old one for reference https://github.com/theforeman/foreman-packaging/pull/10037

Odilhao commented:
Approved, it would nice to get a katello-master-source-release green run before merging this one.

evgeni commented:
> Approved, it would nice to get a katello-master-source-release green run before merging this one.

`s/nice/absolutely necessary/` ;-)

evgeni commented:
[test rpm-copr]
